Integrations/integration-TCPIPUtils.yml

```diff
@@ -0,0 +1,94 @@
+description: enrich information from tcipip.com
+configuration:
+- display: ""
+  name: apiKey
+  defaultvalue: ""
+  type: 4
+  required: true
+script:
+  script: |-
+    import urllib2, json
+
+    API_KEY = demisto.params()["apiKey"]
+
+    IP2ASN_API_URL = "https://www.utlsapi.com/api.php?type=ip2asn&q=%s&version=1.0&apikey=" + API_KEY
+    IPV4_INFO_API_URL = "https://www.utlsapi.com/api.php?type=ipv4info&q=%s&version=1.1&apikey=" + API_KEY
+
+    # The command demisto.command() holds the command sent from the user.
+    if demisto.command() == 'test-module':
+        query = IP2ASN_API_URL % "8.8.8.8"
+        response = json.loads(urllib2.urlopen(query).read())
+        if response['status'] == "succeed":
+            demisto.results('ok')
+        sys.exit(0)
+    if demisto.command() == 'ip':
+        ec = {}
+        def parse_whois_data(whois_lines):
+            result = {}
+            for line in whois_lines:
+                if line.startswith("#"):
+                    continue
+                key_value = line.split(":")
+                if len(key_value) == 2:
+                    key, value = key_value[0], key_value[1]
+                    result[key.strip()] = value.strip()
+            return result
+
+        query = IPV4_INFO_API_URL % demisto.args()["ip"]
+        response = urllib2.urlopen(query)
+        data = json.loads(response.read())["data"]
+        ip_address = data["question"]
+        ip_data = {
+            "Address": ip_address,
+            "ASN": "AS" + str(data["asinfo"]["asn"]),
+            "Geo": {
+                "Country": data["geoinfo"]["country_name"],
+                "Description": data["geoinfo"]["capital"]
+            },
+            "WHOIS": parse_whois_data(data["whois"]["whois"])
+        }
+        ec[outputPaths["ip"]] = [];
+        ec[outputPaths["ip"]].append(ip_data)
+        hm = {
+            "Address": ip_address,
+            "ASN": ip_data["ASN"],
+            "Country": ip_data["Geo"]["Country"],
+            "City": ip_data["Geo"]["Description"]
+        }
+
+        if ip_data["WHOIS"].has_key("Organization"):
+            hm['Organization'] = ip_data["WHOIS"]['Organization']
+        if ip_data["WHOIS"].has_key("NetRange"):
+            hm['NetRange'] = ip_data["WHOIS"]['NetRange']
+
+        demisto.results({"Type": 1, "Contents": data, "ContentsFormat": "json", "EntryContext": ec,
+        "HumanReadable": tableToMarkdown("TCPIPUtils Info for " + ip_address, [hm], "")})
+
+        sys.exit(0)
+
+    # You can use demisto.args()[argName] to get a specific arg. args are strings.
+    # You can use demisto.params()[paramName] to get a specific params.
+    # Params are of the type given in the integration page creation.
+  type: python
+  commands:
+  - name: ip
```

Comment:
Missing description for command, take a look at other ips integrations (like ipinfo)